sdk/samples/iot/generate_certificate.ps1

```diff
@@ -53,10 +53,10 @@ Write-Output "`nSAMPLE CERTIFICATE GENERATED:"
 Write-Output "Use the following command to set the environment variable for the samples:"
 
 if ($IsWindows) {
-  Write-Output "`n`t`$env:AZ_IOT_DEVICE_X509_CERT_PEM_FILE=$(Resolve-Path device_cert_store.pem)"
+  Write-Output "`n`t`$env:AZ_IOT_DEVICE_X509_CERT_PEM_FILE_PATH=$(Resolve-Path device_cert_store.pem)"
```

Comment:
Nice catch!

Any way we can make sure this script continues to work?

Reply:
that is how I found it

Reply:
May final task for the samples is to review documentation with a test run.  The ps1 script is a new add, so has not received this test yet.